.tekton/serverless-openshift-kn-operator-135-push.yaml

```diff
@@ -23,6 +23,8 @@ spec:
         - VERSION=1.35.0
         - GO_BUILDER=brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_golang_1.22
         - GO_RUNTIME=registry.access.redhat.com/ubi8/ubi-minimal
+        - JAVA_BUILDER=
+        - JAVA_RUNTIME=
```

Comment:
hmm

Reply:
Seems this is fixed now (https://github.com/openshift-knative/serverless-operator/compare/b158c943c5bfd813fedadddf42363c35d7e236ba..f1894e351deaaf687566136348ed58fcaaef46b4)